src/plugins/oSnap/components/Input/Amount.vue

```diff
@@ -16,24 +16,31 @@ const dirty = ref(false);
 
 const format = (amount: string) => {
   try {
-    return parseUnits(amount, props.decimals).toString();
+    // empty string throws
+    const parsed = parseUnits(amount, props.decimals).toString();
+    return parsed;
   } catch (error) {
-    return undefined;
+    return parseUnits('0', props.decimals).toString();
```

Comment:
do you actually need to call parse units on 0? cant you just put '0'?

Reply:
also im not sure if we should really be resetting input box to 0 if theres an error